storage/storage.go

```
@@ -0,0 +1,64 @@
+package storage
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+)
+
+const file = "./.storage.json"
+
+type Table struct {
+	GithubAccessToken       string
+	TransferredRepositories []Repo
+	UpdatedRepositories     []string
+}
+
+type Repo struct {
+	OriginalURL string
+	TargetOrg   string
+}
+
+func open() (Table, error) {
+	var t Table
+
+	data, err := ioutil.ReadFile(file)
+	if os.IsNotExist(err) {
+		return t, nil
```

Swallowed possible `ErrNotExist` error.